htscodecs/tokenise_name3.c

```diff
@@ -1464,11 +1468,17 @@ uint8_t *tok3_encode_names(char *blk, int len, int level, int use_arith,
 
     // Encode name
     for (i = j = 0; i < len; j=++i) {
-        while (i < len && blk[i] > '\n')
+        while (i < len && blk[i] >= ' ') // also catches top-bit set
```

Comment:
This won't work on platforms with unsigned char (e.g. ARM).

Reply:
Good spot.  I always forget that there are systems where char can be unsigned!  Fixed.